packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/Update/UpdateStatusActionBarIcon.tsx

```diff
@@ -177,10 +178,6 @@ export const UpdateStatusActionBarIcon = ({
 
     const isDesktopSuite = isDesktop();
 
-    if (device?.features === undefined) {
```

Comment:
abscence of device hit all the toolips / icons, but Suite update could be available